modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: PROCEDURE
+:description: Starting a new workspace from a devfile URL
+:keywords: start-new-workspace, start-a-new-workspace, how-to-start-new-workspace, how-to-start-a-new-workspace, starting-a-new-workspace, how-to-start-workspace, how-to-start-a-workspace
+:navtitle: Starting a new workspace from a devfile URL
+:page-aliases:
+
+[id="starting-a-new-workspace-from-a-devfile-url"]
+= Starting a new workspace from a devfile URL
+
+Working with {prod-short} in your browser involves multiple URLs:
+
+* The URL of your organization's {prod-short} instance, used as part of all the following URLs
+* The URL of the *devfile*
+* [.underline]#The URLs for starting a new workspace#
+* The URLs of your workspaces in use
+
+With {prod-short}, you can visit a *devfile* URL in your browser to start a new workspace.
+
+pass:[<!-- vale RedHat.Spelling = NO -->]
+
+TIP: You can also use the *Git Repo URL ** field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.
+
+pass:[<!-- vale RedHat.Spelling = YES -->]
+
+.Prerequisites
+
+* Your organization has a running instance of {prod-short}.
+* You know the FQDN URL of your organization's {prod-short} instance: `pass:c,a,q[{prod-url}]`.
+
+.Procedure
+
+To start a new workspace from a devfile URL:
+
+. Optional: Visit your {prod-short} dashboard pages to authenticate to your organization's instance of {prod-short}.
+
+. Visit the URL to start a new workspace using the basic syntax:
+[source,subs="+quotes,+attributes,+macros"]
++
+----
+pass:c,a,q[{prod-url}]#__<devfile_url>__
+----
++
+[TIP]
+====
+You can extend this URL with optional parameters:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<devfile_url>__?__<optional_parameters>__ <1>
+----
+<1> See xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].
+====
++
+.A URL for starting a new workspace
+====
+
+`pass:c,a,q[{prod-url}#https://raw.githubusercontent.com/che-samples/cpp-hello-world/main/devfile.yaml]`
+
+====
++
+[TIP]
+====
+You can pass Personal Access Token to the URL to have access to private repositories:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__https://__<token>__@__<host>__/__<path> <1>
+----
+<1> See xref:using-a-git-provider-access-token.adoc[].
+
+This works for GitHub, GitLab, Bitbucket, Microsoft Azure, and other providers that support Personal Access Token.
+====
```

Comment:
great job @vinokurig  :+1:  smth. we discussed with @l0rd today that the PAT in URL feature really shines  when we use it together with the remotes attribute `?remotes={<git-repo-url>}` and configured SSH key:

- user has a private repo with devfile hosted on some unsupported provider e.g. gogs
- user configures ssh key for this provider 
- user starts workspace using the URL with token for basic auth and remote attribute:

`pass:c,a,q[{prod-url}]#__https://__<token>__@__<host>__/__<path>?remotes={<git-repo-url>}`

- SUCCESS: che server resolves devfile + project is clonned correctly after workspace startup

I believe we can document this in a separate PR and introduce dedicated section in the docs as part of https://github.com/eclipse/che/issues/21951